### ISSUE TITLE
Fonts are cut off

### ISSUE THREAD

**Issue by gtirloni** [WhitewaterFoundry/Fedora-Remix-for-WSL#146]
**Describe the bug**
The fonts are cut, it seems the glyph area is smaller than it should be.

I was going to report it as a Windows Terminal issue but I cannot reproduce with Ubuntu.

**To Reproduce**
Just install Fedora in WSL and open a Windows Terminal.

**Expected behavior**
The fonts show normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/5902830/161786970-28e93299-c07e-48e5-83dc-c9279b800510.png)
![image](https://user-images.githubusercontent.com/5902830/161788190-8dc45d2a-7672-46aa-8a7d-206f02f4255d.png)

**Basic Troubleshooting Checklist**

[x] I have searched Google for the error message.
[x] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[x] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[x] I have searched the WLinux issues page: https://github.com/WhitewaterFoundry/WLinux/issues.
[x] I have reset WLinux: Settings->Apps->Apps & features->WLinux->Advanced Options->Reset. 
[x] I have disabled and re-enabled WSL in Windows Features.
[x] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?
Tried various fonts, font sizes, etc.

**WLinux Version**

WSL 2

Windows Subsystem for Linux Update 5.10.102.1

**Windows Build**

10.0.22000 N/A Build 22000

**Comment by crramirez** [comment]
Hello @gtirloni,

Fedora Remix comes with a theme that uses 'Cascadia Code' as the font name, but by default, Windows Terminal assigns Cascadia Mono. Try to change it to Cascadia Mono to see if the problem goes away.

Regards,
Carlos

**Comment by gtirloni** [comment]
I can't tell if it looks better or not.

![image](https://user-images.githubusercontent.com/5902830/161853731-a7ce97da-8895-4719-9eaf-cddc3567b90c.png)

**Comment by crramirez** [comment]
It looks like no. 

Could you run the following command and tell me the value? `wslsys -S`

**Comment by gtirloni** [comment]
I went to Settings -> Apps and uninstall WSL and WSLg Preview. Deleted Ubuntu and Fedora Remix and rebooted. Installed WSL again (`wsl --update`) and reinstall Ubuntu and Fedora. Reset Windows Terminal settings and configured Cascadia Code. It still looks the same to me.

```
$ wslsys -S
Display Scaling: 1
```

My screen is 1920x1080 120hz.

**Comment by crramirez** [comment]
Everything looks fine. Maybe you have custom scaling here?
![image](https://user-images.githubusercontent.com/8397379/161860053-d18dcf52-1613-4729-86b1-6aa86ad9a542.png)

I found two related issues in Windows Terminal:

https://github.com/microsoft/terminal/issues/12810
https://github.com/microsoft/terminal/issues/4591

**Comment by gtirloni** [comment]
Text size was normal but I found Display was set to a custom scaling of 175% 😕 

I set it back to non-custom 100% and everything looks much smaller but it's fine. The issue with WSL is gone.

The only explaination I can come up with is that I was using a 4K monitor many months ago and probably set it to 175% myself and it kept the value for my internal laptop display? That's weird but apparently I never noticed the scaling ever since.

Thanks for your help even though it wasn't even remotely related to Fedora Remix. Really appreciate it.
